weblate_web/models.py

```diff
@@ -507,17 +507,43 @@ def expires(self):
 
     def get_suggestions(self):
         if not self.support_subscriptions.exists():
-            yield "basic", _("Basic support")
+            yield "basic", _(
+                "Basic support"
+            ), "This will give you more"\
+               " of this and that. "\
+               "You can't resist,"\
+               "because it is a huge deal.",\
+                  "img/Support-Basic.svg", "Get more support"
```

Comment:
This (and all similar blocks below) could be better formatted and the texts should be localizable:

```suggestion
            yield (
                "basic", 
                _("Basic support"),
                _(
                    "This will give you more  of this and that. "
                    "You can't resist, because it is a huge deal."
                ),
                "img/Support-Basic.svg", 
                _("Get more support"),
            )
```

Reply:
I formatted it according to Flake8's requirement, it could be, yes! But then it doesn't pass the flake8.
The texts should be localizable, i will fix that for sure!

Reply:
I have done that in the recent commit.